plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -619,6 +511,17 @@ public void testTimestamp(ZoneId sessionZone)
                 .execute(getQueryRunner(), session, trinoCreateAsSelect(session, "test_timestamp"))
                 .execute(getQueryRunner(), session, trinoCreateAsSelect(getSession(), "test_timestamp"))
                 .execute(getQueryRunner(), session, trinoCreateAndInsert(session, "test_timestamp"));
+    }
+
+    /**
+     * Read timestamp data inserted into by MySql as {@code TIMESTAMP}
+     */
+    @Test(dataProvider = "sessionZonesDataProvider")
+    public void testTimestampFromMySql(ZoneId sessionZone)
+    {
+        Session session = Session.builder(getSession())
+                .setTimeZoneKey(TimeZoneKey.getTimeZoneKey(sessionZone.getId()))
+                .build();
 
         // TODO merge this into the above SqlDataTypeTest once we support writing and reading timestamps with precisions other than 3 (TODO https://github.com/trinodb/trino/issues/6910)
```

Comment:
Move this comment into the javadoc now. And change `into the above` to `testTimestampFromTrino`.

Reply:
Based on [this comment](https://github.com/trinodb/trino/pull/7166#discussion_r588746263), I've dropped this comment entirely. We don't need it to remind us that we'll need more tests when we support more precisions.